packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -732,6 +732,14 @@ wavs = chat.infer(texts, )
 torchaudio.save("output1.wav", torch.from_numpy(wavs[0]), 24000)`,
 ];
 
+export const videocrafter = (): string[] => [
+	`Install from https://github.com/AILab-CVC/VideoCrafter
+
+# Download pretrained T2V models via Hugging Face,
+# and put the model.ckpt in checkpoints/base_512_v2/model.ckpt
+sh scripts/run_text2video.sh`,
+];
+
```

Comment:
Let's focus on download count maybe? I feel that this snippet only works for text2video models but not image2video, which is half of the models if I understand correctly.

Or otherwise you could have the correct script depending on the model id. The checkpoints folder would have to be adapted I guess (`base_512_v2` feels very specific). In that case, I would put a complete bash script. Something like this:

```sh
# Clone repo
git clone https://github.com/AILab-CVC/VideoCrafter
cd VideoCrafter

# Download checkpoint
huggingface-cli download VideoCrafter/VideoCrafter2 model.ckpt --local-dir ./checkpoints/base_512_v2

# Run
sh scripts/run_text2video.sh
```

(to be adapted depending on model id)

Reply:
True, I'll remove the snippet for now